Random Users/View Controller/UsersTableViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  UsersTableViewController.swift
+//  Random Users
+//
+//  Created by Norlan Tibanear on 9/11/20.
+//  Copyright © 2020 Erica Sadun. All rights reserved.
+//
+
+import UIKit
+
+class UsersTableViewController: UITableViewController {
+    
+    
+    // MARK: - Properties
+    var userController = UserController()
+    var users: [UserResults] = [] {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+    
+    private let cache = Cache<String, Data>()
+    private let imageQueue = OperationQueue()
+    var operation = [String: Operation]()
```

Comment:
Arrays should usually be plural, so this could be operations, or better yet fetchImageOperations which gives more insight into what is stored here. Also, you could make note of what the string key is in a comment.